test/instrumentation/testdata/stable-metrics-list.yaml

```diff
@@ -88,6 +88,35 @@
   help: The count of hidden metrics.
```

Comment:
@tallclair all of this was autogenerated from the metrics. I guess beta metrics are actually documented so I'm not sure if I can/will change anything in this file. I can update the comments for the metrics but not sure about that.